### PR TITLE
combine product and proxy exports

### LINKED ISSUE #211: registry-connect: combine products and proxies

### PR DESCRIPTION
**Pull request by theganyo**
Merges "registry-connect apigee product" and "registry-connect apigee proxy" commands and outputs into a single "registry-connect apigee" command and output.

Fixes #211

### CODE REVIEW

**Review by timburks — APPROVED** [review]
Thanks! For the release summaries, could you summarize the changes here or in the description of this issue or #211?

I see that the two subcommands are merged, and I see there's some linking from products to proxies. I'm guessing we might tweak the format of those links, but this seems to match our  overall direction.

**Comment by theganyo** [comment]
Sure, I changed the description to reflect that this just merges the two commands. The links aren't new, and yes, they'll likely need tweaking as we determine how we're going to proceed with signals (see also https://github.com/apigee/registry-experimental/issues/214).
